pkg/resource/resource_types.go

```diff
@@ -143,18 +143,19 @@ var supportedTypes = map[string]ResourceTypeMeta{
 		"aws_apigatewayv2_route",
 		"aws_apigatewayv2_integration",
 	}},
-	"aws_apigatewayv2_route":          {},
-	"aws_apigatewayv2_vpc_link":       {},
-	"aws_apigatewayv2_authorizer":     {},
-	"aws_apigatewayv2_integration":    {},
-	"aws_apigatewayv2_model":          {},
-	"aws_apigatewayv2_stage":          {},
-	"aws_apigatewayv2_route_response": {},
-	"aws_apigatewayv2_deployment":     {},
-	"aws_apigatewayv2_domain_name":    {},
-	"aws_launch_template":             {},
-	"aws_launch_configuration":        {},
-	"aws_apigatewayv2_api_mapping":    {},
+	"aws_apigatewayv2_model":                {},
+	"aws_apigatewayv2_stage":                {},
+	"aws_apigatewayv2_route_response":       {},
+	"aws_apigatewayv2_deployment":           {},
+	"aws_apigatewayv2_domain_name":          {},
+	"aws_launch_template":                   {},
+	"aws_launch_configuration":              {},
```

Comment:
Can you put these two `aws_launch` resources at the end? Let's regroup all v2 types.